Documentation/configuration/sbom.rst

```diff
@@ -31,22 +31,32 @@ Prerequisites
 Download SBOM
 =============
 
-The SBOM can be downloaded from the supplied Cilium image using the
-``cosign download sbom`` command.
+You can download the SBOM in-toto attestation from the supplied Cilium image using the following command:
 
 .. code-block:: shell-session
 
-    $ cosign download sbom --output-file sbom.spdx <Image URL>
+    $ cosign download attestation --predicate-type spdxjson <Image URI> | jq -r .payload | base64 -d | jq .predicate > ciliumSBOM.spdx.json
 
-Verify SBOM Image Signature
-===========================
+Verify SBOM attestation
+=======================
 
-To ensure the SBOM is tamper-proof, its signature can be verified using the
-``cosign verify`` command.
+To verify the SBOM in-toto attestation on the supplied Cilium image, run the following command:
 
 .. code-block:: shell-session
 
-    $ COSIGN_EXPERIMENTAL=1 cosign verify --certificate-github-workflow-repository cilium/cilium --certificate-oidc-issuer https://token.actions.githubusercontent.com --attachment sbom <Image URL> | jq
+    $ TAG=v1.15.0
```

Comment:
I noticed that this doesn't work for the v1.15.0 TAGs. Can we change this so the value is automatically updated whenever we generate a new tag starting from 1.16 pre-release? cc @qmonnet

Reply:
So we could have something like this:

```
.. parsed-literal::

    $ TAG=|CURRENT_RELEASE|
    $ cosign ...
```

But this would pick up the version from file `VERSION`, which does not correspond to an existing tag on `main`.

I'm not sure how we'd update this to pick up the latest tag. I've struggled more than I expected, recently, to figure out a way to pick up the “latest release” tag for the upgrade/downgrade workflows in CI. The issue is that there's no easy way to tell what's the relevant tag; and it also requires the Git history to be available.

Or did you have a specific solution in mind? Or would just using `|CURRENT_VERSION|` work for the docs? After all, we don't need all tags to appear in the docs?

Reply:
> Or did you have a specific solution in mind? Or would just using |CURRENT_VERSION| work for the docs? After all, we don't need all tags to appear in the docs?

@qmonnet I meant the same thing we do in 
![image](https://github.com/cilium/cilium/assets/5714066/6c8c3446-d0cb-45f5-82b4-ff866a718943)

from https://docs.cilium.io/en/v1.15/gettingstarted/k8s-install-default/

Reply:
OK so that's:

```
       .. parsed-literal::

          cilium install |CHART_VERSION|
```

With `chart_version = '--version ' + release`; so that's likely `|IMAGE_TAG|` that we'd need, given that Documentation/conf.py defines `image_tag = 'v' + release`.